src/components/BottomBar.js

```diff
@@ -67,12 +67,11 @@ function BottomBar ({programStarted, setProgramStarted}) {
       let value;
       window.CrComLib.subscribeState('s', `${index + 91}`, incomingValue => {
```

Comment:
We want to avoid '[magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))' like `91`. In this case, consider creating a constant name after what the number represents it and use it inline. Eg: 
`const CAMERA_BASE_IDX = 91;` and then use as `${index + CAMERA_BASE_IDX }`